### PR TITLE
Review

### PR DESCRIPTION
**Pull request by Nasser-Eddin-Nasser**
Fully review for the Backend-Server and the HR-Monitor!

### CODE REVIEW

**Comment by Nasser-Eddin-Nasser** [comment]
@dalex135  and I had a pair-programming, we tested every line in the code and ran it. It has been refactored and now it looks much better than before :-)

File: Backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 spring.jpa.database=POSTGRESQL
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 spring.jpa.show-sql=true
-spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.generate-ddl=true 
+spring.jpa.hibernate.ddl-auto=create-drop
```
Comment (vishwasanavatti) [inline Backend/src/main/resources/application.properties:9]: why is it create-drop? it should be update as we dont want to lose data on restart

File: HR-Monitor/src/Main/Configuration.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import java.nio.file.Paths;
 
 public class Configuration {
-    public static final String BES_URI = "http://localhost:9090/api/HRController/";
+    public static final String Backend_Server_URL = ""; //  Http or Https
```
Comment (vishwasanavatti) [inline HR-Monitor/src/Main/Configuration.java:7]: should be https. this change is there in my PR.

**Review by vishwasanavatti — COMMENTED** [review]
Looks good. I have mentioned 2 changes and will approve the PR after checking alignment.

**Review by vishwasanavatti — CHANGES_REQUESTED** [review]
Screenshot 2020-07-18 at 11 42 25 PM

I found few issues as in screenshot. Description in work experience is not shown fully. It should be adjusted according to text length. 
Most importantly, why are images being deleted on logout? When I logout and open the application again, image is not displayed anymore.

**Comment by Nasser-Eddin-Nasser** [comment]
@vishwasanavatti  the GUI has resize function, so it depends on your screen resolution and size, you only need to resize it and everything should be readable and fine.
By log-out or close the Application, the Filesystem will be cleaned up. Here it can be a small bug after logging in, ill check it and update the PR. 
Thanks for the review :)

**Comment by Nasser-Eddin-Nasser** [comment]
> @vishwasanavatti the GUI has resize function, so it depends on your screen resolution and size, you only need to resize it and everything should be readable and fine.
> By log-out or close the Application, the Filesystem will be cleaned up. Here it can be a small bug after logging in, ill check it and update the PR.
> Thanks for the review :)

there is no Bug, I just test it, the Filesystem of the BES should be different than the one in the HRM.

**Comment by vishwasanavatti** [comment]
> @vishwasanavatti the GUI has resize function, so it depends on your screen resolution and size, you only need to resize it and everything should be readable and fine.

Telefonnumer is not aligned when screen is small or when I maximise it as well. the letter "r" still comes in next line. should be a small fix here. 
The description in work experience is a text area and it can be 2-3 lines or even more. how can you resize to see it all? can you resize and put a screenshot ?
Also I found one small issue, when we fill data to create account. The data has to cleared after creation.

> By log-out or close the Application, the Filesystem will be cleaned up. Here it can be a small bug after logging in, ill check it and update the PR.

Its not a bug. Path should be different. updated in README file about this.

File: Backend/src/main/resources/application.properties (hunk 1 above)
Comment (Nasser-Eddin-Nasser) [reply Backend/src/main/resources/application.properties:9]: Default config, the user can change it!

File: HR-Monitor/src/Main/Configuration.java (hunk 2 above)
Comment (Nasser-Eddin-Nasser) [reply HR-Monitor/src/Main/Configuration.java:7]: Users can run the HRM also in HTTP! 
personal configuration of the users!

**Comment by Nasser-Eddin-Nasser** [comment]
> > @vishwasanavatti the GUI has resize function, so it depends on your screen resolution and size, you only need to resize it and everything should be readable and fine.
> 
> Telefonnumer is not aligned when screen is small or when I maximise it as well. the letter "r" still comes in next line. should be a small fix here.
> The description in work experience is a text area and it can be 2-3 lines or even more. how can you resize to see it all? can you resize and put a screenshot ?
> Also I found one small issue, when we fill data to create account. The data has to cleared after creation.
> 
> > By log-out or close the Application, the Filesystem will be cleaned up. Here it can be a small bug after logging in, ill check it and update the PR.
> 
> Its not a bug. Path should be different. updated in README file about this.

image
image

**Review by vishwasanavatti — APPROVED** [review]

**Pull request merged**